skyportal/utils/thumbnail.py

```diff
@@ -0,0 +1,38 @@
+from PIL import Image, ImageStat, UnidentifiedImageError
+
+
+def image_is_grayscale(file, thumb_size=40, MSE_cutoff=22, adjust_color_bias=True):
```

Comment:
Please make these configurable in `config.yaml` and grab them in the calls from there so that we could tweak things easily if necessary.